drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -479,6 +483,25 @@ func (k *k8sSchedOps) IsPXReadyOnNode(n node.Node) bool {
 	return true
 }
 
+// IsPXInstalled returns true/false if px label is enabled/disabled on kubernetes node
+func (k *k8sSchedOps) IsPXInstalled(n node.Node) bool {
+	isPxInstalled := true
+	kubeNode, err := k8s.Instance().GetNodeByName(n.Name)
+	if err != nil {
+		logrus.Errorf("Failed to get node %v", err)
+		return false
+	}
+
+	// if node has px/enabled label set to false or node-type public or
+	// has any taints then make above assumption wrong
+	if kubeNode.Labels[pxenabled] == "false" || kubeNode.Labels[nodeType] == "public" || len(kubeNode.Spec.Taints) > 0 {
```

Comment:
@harsh-px do we want to return false on any taint? or only for NoSchedule/PreferNoSchedule?